### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by tikohov20**
Basically same, but better.

( also didn't test )

### CODE REVIEW

**Review by HrachAsatryan — CHANGES_REQUESTED** [review]
Rewrite in Python

File: VoiceCommands/connection.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+        connection.subscribe(player);
+
+        return connection;
+      }
+    }
+  }
+}
+
+module.exports = connection;
```
Comment (vardgesmusakhanyan) [inline VoiceCommands/connection.js]: We could write this section in Python

File: VoiceCommands/queue.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      if (player.state.status === AudioPlayerStatus.Idle) {
+        this.playNextInQueue();
+      }
+    },
+    reset() {
+      queue = []
+    },
+    playNextInQueue() {
+      const stream = getAudioStreamFromUrl(queue[0].url);
+      console.log(queue[0]);
```
Comment (tikohov20) [inline VoiceCommands/queue.js]: remove console log

**Review by tikohov20 — COMMENTED** [review]
add masterpiece.mp3 as a fallback song

**Comment by HrachAsatryan** [comment]
Please add check, if video URL is https://youtu.be/TUVcZfQe-Kw replace with https://youtu.be/WHuBW3qKm9g

**Comment by tikohov20** [comment]
> Rewrite in Python

image

**Pull request merged**